### DISCUSSION TITLE
error trying to import CLSIDFromProgID

### DISCUSSION THREAD

**Discussion by notwopr** [mhammond/pywin32#2523] [General]
Note that issues in this repository are only for bugs or feature requests in the pywin32.

**If you need support or help using this package, please follow [these instructions](https://github.com/mhammond/pywin32/blob/master/README.md#support)** - support or help requests will be closed without comment.

For all bugs, please provide the following information.

* Expected behavior and actual behavior.
I thought CLSIDFromProgID is a function defined in the win32com.client module of the PyWin32 package, which provides access to the COM API in Windows. The function is used to obtain the CLSID of a given ProgID.

* Steps to reproduce the problem.
when I do the following import:
`from win32com.client import CLSIDFromProgID`
I get the following error:
`ImportError: cannot import name 'CLSIDFromProgID' from 'win32com.client' (C:\Program Files\Python311\Lib\site-packages\win32com\client\__init__.py)`

* Version of Python and pywin32
3.11.2
pywin32 vers 306

**Comment by Avasam** [comment]
I can confirm `CLSIDFromProgID` is not currently exposed to the Python API. It's wrapped in `pywintypes.IID` which is a method that returns `pywintypes.IIDType`

```py
>>> import pywintypes
>>> pywintypes.IID("msxml2.domdocument")
IID('{F6D90F11-9C73-11D3-B32E-00C04F990BB4}')
>>> str(pywintypes.IID("msxml2.domdocument"))
'{F6D90F11-9C73-11D3-B32E-00C04F990BB4}'
>>> type(pywintypes.IID("msxml2.domdocument"))
<class 'PyIID'>
>>> pywintypes.IIDType
<class 'PyIID'>
```
